daisy_workflows/image_build/enterprise_linux/kickstart/rocky_linux_8_optimized_gcp_accelerator.cfg

```diff
@@ -187,7 +187,7 @@ dnf install -y google-cloud-cli
 dnf install -y https://depot.ciq.com/public/files/gce-accelerator/open-gpu-kernel-modules-el8-x86_64/nvidia-open-gpu-repos.noarch.rpm https://depot.ciq.com/public/files/gce-accelerator/open-gpu-kernel-modules-el8-x86_64/nvidia-mellanox-ofed-repos.noarch.rpm
 dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel8/x86_64/cuda-rhel8.repo
 
-dnf install -y nvidia-open-gpu-kernel-modules nvidia-accelerated-graphics-driver mlnx-ofed-guest
+dnf install -y nvidia-dc-driver-latest-cuda kmod-nvidia-dc-open-latest doca-ofed
```

Comment:
Please include the details in the PR as to why the packages are changing.

Reply:
Updated the description